docs/build-bot.md

```diff
@@ -136,6 +171,58 @@ Note that other platforms are still 'skip' but not included in the build set.
 Finally, the PR author pushes another commit, with `Build-bot: release windows`.
 The build set is now: `(android:skip ios:skip windows:release)`.
 
+# FAQ
+
+* How do I specify commands for multiple platforms, e.g. building on Windows and Linux?
+
+```
+Build-bot: build windows,linux
+```
+
+* If I modify a file that would cause a build on all platforms, does `Build-bot:
+  build android` then cause a build only on Android and skip all others? Or is
+  it only additive?
+
+The commands are additive. To skip all others, you would do:
+
+```
+Build-bot: skip
+Build-bot: build android
+```
+
+* What happens in the following scenario: I have a PR that I only want to build
+  on Android and I add `Build-bot: build android`. Later on I merge in the
+  changes from `master` and add another commit. If I still only want to build on
+  Android, do I have to add `Build-bot: build android` on my new commit again?
+  Or what happens with merges that touch other files which would cause builds
+  for other platforms?
+
+First, you would need to specify `skip` for all other platforms in order to only
+build on Android:
+
+```
+Build-bot: skip
+Build-bot: build android

```

Comment:
I wish we'd have made this more similar to the builder scripts (where we have `<actions>:[targets]`) and made this `<BuildLevel>:[Platforms]`. That would allow to write this in one line: `Build-bot: skip build:android`